ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/properties/BorderStyle.kt

```diff
@@ -24,3 +52,15 @@ internal fun rememberBorderStyle(border: Border): BorderStyle {
         color = colorStyle,
     )
 }
+
+@Composable
+@JvmSynthetic
+internal fun rememberBorderStyle(border: BorderStyles): BorderStyle {
```

Comment:
Do we still need the remember method that accepts a `Border` after this? I see it's only used for some previews, but we could change to only use this?

Reply:
Ohh I see this was done in a follow up PR. Nvm then! :)